Keda.Cosmosdb.Scaler/src/Startup.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Keda.CosmosDB.Scaler.Repository;
+using Keda.CosmosDB.Scaler.Services;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Keda.CosmosDB.Scaler
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddGrpc();
+            services.AddSingleton<ICosmosDBRepository, CosmosDBRepository>();

```

Comment:
I don't think we need a singleton dependency?
```suggestion
            services.AddTransient<ICosmosDBRepository, CosmosDBRepository>();
```

Reply:
The repository has an inmemory cache of builder objects which requires a singleton instance.